assets/src/edit-story/components/library/text/fontPreview.js

```diff
@@ -29,27 +28,34 @@ import { useFont } from '../../../app';
 import { ALLOWED_EDITOR_PAGE_WIDTHS, PAGE_WIDTH } from '../../../constants';
 import StoryPropTypes from '../../../types';
 import stripHTML from '../../../utils/stripHTML';
+import { Text, themeHelpers } from '../../../../design-system';
 
 const PREVIEW_EM_SCALE = ALLOWED_EDITOR_PAGE_WIDTHS[0] / PAGE_WIDTH;
 
 const Preview = styled.button`
-  background: ${({ theme }) =>
-    rgba(theme.DEPRECATED_THEME.colors.fg.white, 0.1)};
-  padding: 12px 16px;
-  margin-bottom: 12px;
-  border-radius: 4px;
+  background-color: ${({ theme }) =>
+    theme.colors.interactiveBg.secondaryNormal};
+  padding: 8px 16px;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
   width: 100%;
   border: none;
   cursor: pointer;
   text-align: left;
+
+  :hover {
+    background-color: ${({ theme }) =>
+      theme.colors.interactiveBg.secondaryHover};
+  }
+
+  ${themeHelpers.focusableOutlineCSS};
 `;
 
-const Text = styled.span`
-  background: none;
+const PreviewText = styled(Text).attrs({ forwardedAs: 'span' })`
+  color: ${({ theme }) => theme.colors.fg.primary};
   font-size: ${({ fontSize }) => fontSize * PREVIEW_EM_SCALE}px;
   font-weight: ${({ fontWeight }) => fontWeight};
   font-family: ${({ fontFamily }) => fontFamily};
```

Comment:
@barklund the designs have this as Google Sans, but we're currently seeing Roboto.

https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/library/panes/text/textPresets.js#L76-L91

Looks like this is inheriting from the global font family. Is there a plan to update that globally? Should we just update the font family here for now?

Reply:
There's an issue for the font change: https://github.com/google/web-stories-wp/issues/6386